binzx/otomi

```diff
@@ -51,6 +51,7 @@ readonly base_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
 
 packageIsCore() {
   path=$1
+  [[ $path == "/home/app/stack"* ]] && return 1
   [ ! -f "$path/package.json" ] && return 1
```

Comment:
```suggestion
  { [ $path = "/home/app/stack" ] || [ ! -f "$path/package.json" ]; } && return 1
```

Reply:
seeing we only invoke this function from one location I would think we don't even need the function, but if you keep it, can you also please rename it to snake_case format?